understanding/22/focus-appearance-minimum.html

```diff
@@ -22,7 +22,7 @@ <h2>Focus Appearance (Minimum) Success Criteria text</h2>
         
             <p>For the keyboard focus indicator of each User Interface Component, all of the following are true:</p>
             <ul>
-                <li><strong>Minimum area:</strong> The <a>focus indication area</a> is greater than or equal to a 1 <a>CSS pixel</a> border of the focused control, or has a thickness of at least 8 CSS pixels along the shortest side of the element.</li>
+                <li><strong>Minimum area:</strong> The <a>focus indication area</a> is greater than or equal to a 1 <a>CSS pixel</a> border of the focused control, or the focus indication area a thickness of at least 8 CSS pixels along the shortest side of the element.</li>
```

Comment:
missing a verb now.

"or the focus indication area a thickness" > "or the focus indication area has a thickness"